ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/row.view.js

```diff
@@ -82,6 +82,15 @@ module.exports = Marionette.LayoutView.extend({
         var prefs = user.get('user').get('preferences');
         var preferredHeader = user.get('user').get('preferences').get('columnOrder');
         var hiddenColumns = user.get('user').get('preferences').get('columnHide');
+
+        var resultTemplateAttributes = [];
+        if (store.getCurrentQuery()) {
+            var selectedResultTemplate = store.getCurrentQuery().get('selectedResultTemplate');
```

Comment:
❗️ Similar comment about accessing a model property that isn't defined in [Query.js](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/js/model/Query.js#L74-L88)